wazo_call_logd/plugins/retention/listener.py

```diff
@@ -0,0 +1,15 @@
+from wazo_call_logd.database.queries.retention import RetentionDAO
```

Comment:
Have you installed the [copyright git hook](https://github.com/wazo-platform/wazo-tools/blob/master/git-hooks/copyright-check)? Because you forgot to add the copyright header :smiley: